Gopkg.toml

```diff
@@ -31,7 +31,8 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/keycloak-client"
-  version = "v1.0-rc8"
+  #version = "v1.0-rc7"
+  branch="multi-token"
```

Comment:
Remove before merging ? (do we keep it in a branch while production is still in 6.0.1 ?)

Reply:
We will merge in master.